pkgs/top-level/aliases.nix

```diff
@@ -1186,6 +1186,7 @@ mapAliases ({
   ncat = throw "'ncat' has been renamed to/replaced by 'nmap'"; # Converted to throw 2022-02-22
   neap = throw "neap was removed from nixpkgs, as it relies on python2"; # Added 2022-01-12
   neochat = libsForQt5.kdeGear.neochat; # added 2022-05-10
+  neofetch = throw "neofetch was removed: upstream had not updated it since 2021-12-10. Seek for alternative fetchers, like afetch."; # Added 2023-09-16
```

Comment:
Looks like the version of `afetch` in nixpkgs is even older than `neofetch`?

Reply:
Seriously?
I was thinking in a minimal, C-only or shell-only fetcher, but the one I found is older?

Is there any options at all?

Reply:
Let's go with hyfetch